cocos/core/gfx/base/define.ts

```diff
@@ -2412,3 +2412,8 @@ export function formatAlignment (format: Format): FormatAlignment {
 export function alignTo (size: number, alignment: number): number {
     return Math.ceil(size / alignment) * alignment;
 }
+
```

Comment:
This file is auto generated. So it has eslint problem. It will be fixed in the future.